File: PRAK110-2510817220008-NamiraHumaira.py
import math
al = 12
ti = 5
sa = ti
sb = math.sqrt(al * al + ti * ti) 
sc = al 
kel = (sa + sb + sc)
lu = (al * ti) / 2  
print("Diketahui :")
print("Alas = {} cm " .format(al)) 
print("Tinggi = {} cm" .format(ti)) 
print("")
print("Jawab :")
print("Sisi A = {} cm" .format(ti)) 
print("Sisi B = {} cm" .format(round(sb)))
print("Sisi C = {} cm" .format(al))
print("Keliling = {} cm" .format(round(kel)))
print("Luas = {} cm" .format(round(lu)))

File: PRAK110-2510817220008-NamiraHumaira.c
#include <stdio.h> 
#include <math.h>

int main()
{
    int ti = 12; 
    int al = 5; 
    int sa = al;
    int sb = sqrt(ti * ti + al * al);
    int sc = ti;
    int kel= (sa + sb + sc); 
    int lu = (al * ti) / 2;
    printf("Diketahui :"); 
    printf("\n");
    printf("Alas = %d cm\n", al);
    printf("tinggi = %d cm\n", ti); 
    printf("\n");
    printf("Jawab : "); 
    printf("Sisi A = %d cm\n", ti); 
    printf("Sisi B = %d cm\n", sb); 
    printf("Sisi C = %d cm\n", al);
    printf("Keliling = %d cm\n", kel); 
    printf("Luas = %d cm\n", lu); 
    return 0;
}